src/basic/conf-files.h

```diff
@@ -47,3 +47,9 @@ int conf_file_read(
                 void *userdata,
                 bool ignore_enoent,
                 bool *invalid_config);
+
+#define NETWORK_DIRS ((const char* const*) CONF_PATHS_STRV("systemd/network"))
+#define NETWORK_DIRS_NULSTR CONF_PATHS_NULSTR("systemd/network")
```

Comment:
maybe introduce a new network-util.h for this? would be tiny for now, but I'd still do that.